README.rst

```diff
@@ -114,14 +114,14 @@ example
     i = w.into(
         lambda price, doctor, patient_name: {
             "doctor": doctor,
-            "patients": [patient_name],
+            "patients": patient_name,
             "cost": price,
         }
     )
     a = i.aggregate("$.['cost', 'patients']")
```

Comment:
do you not also want the option of an aggregate function where if present it reduces the arrays like you originally had with sum?

Reply:
@ldomic you and @MattOates  have opposite, but equally valid,  ideas here....... worth a chat?

Reply:
Should the easiest way be to create a new separate function so that you can choose what it is you need?

Reply:
i'd like to keep the number of functions down to a bare minimum if possible. could the aggregation function be an optional argument?

Reply:
I guess you mean have an optional argument? `sum=True`?